spec/requests/api/tree_spec.rb

```diff
@@ -9,6 +8,8 @@
       get "/api/#{version.project.platform}/#{version.project.name}/tree?api_key=#{user.api_key}"
       expect(response).to have_http_status(:success)
       expect(response.content_type).to eq('application/json')
+
+      version.reload
```

Comment:
Why does this need to reload the version after a `GET`?

Reply:
The create generates a closure which can't be dumped into the cache, maybe I can use let! rather than let to achieve the same thing, I'll try that...

Reply:
Shockingly, it doesn't work. Using reload make it work, but using let! doesn't.

Reply:
That makes me think there's something which isn't getting serialized right or pulled out of the db fully. 

Also, I suspect we actually don't want to be using the cached version in the test -- otherwise we're not comparing the new generation code

Reply:
The test config is on a nullstore which always results in a miss. For some reason it still attempts to serialize it though. The FactoryBot doesn't always appear to fully load things from the DB.